### ISSUE TITLE
Error while moving the mouse in Internet explorer, causing significant performance issue

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#11037]
### Steps to reproduce:
1. Open any website in Internet explorer
2. Move the mouse quickly over the page

### Actual behavior:
Following errors are written to the log:
```
DEBUGWARNING - virtualBuffers.VirtualBuffer._isNVDAObjectInApplication_noWalk (21:54:42.199) - MainThread (14084):
getIdentifierFromNVDAObject failed. Object probably died while walking ancestors.
Traceback (most recent call last):
  File "virtualBuffers\__init__.pyc", line 709, in _isNVDAObjectInApplication_noWalk
  File "virtualBuffers\MSHTML.pyc", line 235, in getIdentifierFromNVDAObject
LookupError
```

and

```
WARNING - eventHandler._EventExecuter.next (22:00:55.707) - MainThread (10196):
Could not execute function event_mouseMove defined in NVDAObjects module; kwargs: {'x': 1314, 'y': 1084}
Traceback (most recent call last):
  File "eventHandler.pyc", line 100, in next
  File "NVDAObjects\__init__.pyc", line 1063, in event_mouseMove
  File "NVDAObjects\IAccessible\MSHTML.pyc", line 427, in makeTextInfo
  File "documentBase.pyc", line 24, in makeTextInfo
  File "NVDAObjects\IAccessible\MSHTML.pyc", line 333, in __init__
TypeError: not all arguments converted during string formatting
ERROR - eventHandler.executeEvent (22:00:55.707) - MainThread (10196):
error executing event: mouseMove on <NVDAObjects.Dynamic_EditableTextWithoutAutoSelectDetectionMSHTMLButtonIAccessible object at 0x0779AC30> with extra args of {'x': 1314, 'y': 1084}
Traceback (most recent call last):
  File "eventHandler.pyc", line 100, in next
  File "NVDAObjects\__init__.pyc", line 1063, in event_mouseMove
  File "NVDAObjects\IAccessible\MSHTML.pyc", line 427, in makeTextInfo
  File "documentBase.pyc", line 24, in makeTextInfo
  File "NVDAObjects\IAccessible\MSHTML.pyc", line 333, in __init__
TypeError: not all arguments converted during string formatting

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "eventHandler.pyc", line 155, in executeEvent
  File "eventHandler.pyc", line 92, in __init__
  File "eventHandler.pyc", line 107, in next
  File "extensionPoints\util.pyc", line 170, in callWithSupportedKwargs
  File "NVDAObjects\__init__.pyc", line 1063, in event_mouseMove
  File "NVDAObjects\IAccessible\MSHTML.pyc", line 427, in makeTextInfo
  File "documentBase.pyc", line 24, in makeTextInfo
  File "NVDAObjects\IAccessible\MSHTML.pyc", line 333, in __init__
TypeError: not all arguments converted during string formatting
DEBUG - speech.manager.SpeechManager._handleIndex (22:00:55.768) - MainThread (10196):
Unknown index 283, speech probably cancelled from main thread.
```

the unknown index might be the cause here for the performance lag, about 6 or 7 seconds until NVDA says anything:
```
IO - inputCore.logTimeSinceInput (22:00:55.958) - MainThread (10196):
6.260 sec since input
IO - speech.speak (22:00:55.958) - MainThread (10196):
Speaking ['Abschnitt']
IO - inputCore.logTimeSinceInput (22:00:55.967) - MainThread (10196):
6.270 sec since input
IO - speech.speak (22:00:55.967) - MainThread (10196):
Speaking ['\r\n\r\n\r\nWir setzen auf Werbung, damit wir kostenlose Inhalte für Sie bereitstellen können.\r\n\r\nDurch Klicken auf einen beliebigen Inhalt der Seite oder auf die Schaltfläche “Weiter zur Webseite”, erklären Sie sich damit einverstanden, dass wir und verschiedene andere Anbieter Cookies auf Ihrem Gerät für MSN und andere Microsoft News Websites speichern und abrufen können. Diese Cookies werden für Analysen, Inhalte und Werbung verwendet. Einige Beispiele enthalten geografische Daten zu Leistungs- und Personalisierungszwecken sowie für interessenbasierte Werbung. Sie erhalten weitere Informationen zu Werbezwecken oder zu Anbietern, die wir verwenden, indem Sie weiter unten auf "Anzeigeneinstellungen verwalten" klicken. Die Einstellungen können jederzeit geändert werden, indem Sie in der Fußzeile auf den Link "Datenschutzeinstellungen" klicken. Weitere InformationenAnbieter anzeigen \r\n\r\n\r\n\r\n\r\nClose\r\n\r\n\r\n\r\nWeiter zur Webseite\r\n\r\n\r\n\r\nEinstellungen für Werbeanzeigen verwalten']
IO - inputCore.logTimeSinceInput (22:00:56.788) - MainThread (10196):
7.090 sec since input
IO - speech.speak (22:00:56.788) - MainThread (10196):
Speaking ['Link']
IO - inputCore.logTimeSinceInput (22:00:56.788) - MainThread (10196):
7.090 sec since input
IO - speech.speak (22:00:56.788) - MainThread (10196):
Speaking [' \r\nDie größten Fußballstadien der Welt\r\nran.de  ']
```

I think NVDA actually does not know where the mouse cursor is and tries to report as much as possible. I have the feeling that this might be a regression, but no idea when this issue started.

### Expected behavior:
No errors, mouse navigation should work like in all other browsers.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-20015,56a62097

#### Windows version:
Windows 10 1909 Update

#### Name and version of other software in use when reproducing the issue:
Internet explorer 11 last version

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes, 2019.3.1, same issue.

#### If addons are disabled, is your problem still occuring?
yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
yes, no effect.

**Comment by lukaszgo1** [comment]
@Adriani90 I believe this should be fixed in #11050. Can you confirm?

**Comment by Adriani90** [comment]
Yes fixed now. Thanks very much.
Closing as works for me.
